scss/_patterns_article-pagination.scss

```
@@ -2,11 +2,12 @@
 
 @mixin vf-p-article-pagination {
   %chevron-icon {
-    color: $color-mid-dark;
-    content: '\203A';
-    font-size: 2em;
+    @extend %icon;
+    @include vf-icon-chevron($color-mid-dark);
+
+    content: '';
     position: absolute;
-    top: 1rem;
+    top: 1.25rem;
```

This component is quite dated an doesn't use our latest conventions, but it still would be best to express that using our spacing variables, I'd suggest below:

```suggestion
    top: calc($spv--large + $spv--x-small);
```